content/resources/csc290.md

```diff
@@ -0,0 +1,46 @@
+---
+title: CSC290
+icon: /icons/csc290.svg
+desc: Data Structures and Analysis
+link: csc290
+tags:
+  - utm-cs
+---
+
+# CSC290: Communication Skills for Computer Scientists
+
+#### Prerequisite: [CSC148](./csc148)
+
+<grid-1-x-2 title="Fall 2019 Class Website" img-src="https://i.imgur.com/1APtR8j.png" link="https://www.cs.toronto.edu/~lczhang/290/" desc="All credits to Lisa Zhang" button="Check it out!"></grid-1-x-2>
+
+<ExamText class-code="CSC290"></ExamText>
+
+### What is CSC290?
+
+CSC290 is a computer science course focused on improving student's communication
+skills in both inperson and over the computer. Students will tackle everyday
+problems that computer scientists face outside of programming.
+
+After this course, Students will be able compose formal emails, write cover
+letters and lead software projects.
+
+### Topics in CSC290:
+
+- Communication Theory
+- Critical Thinking
+- Project Management
+- Interpersonal Communication
+- Interpersonal Scenarios
+- Technical Writing
+- Resume and Cover Letters
+- Technical Interviews
+
+### Resources:
+
+<grid-1-x-2 title="Improve your Interpersonal Skills with Google" :reversed="true" img-src="https://www.datocms-assets.com/7756/1579108013-interpersonal-skills-1.png?w=1000&q=70&auto=compress,format,enhance" link="https://learndigital.withgoogle.com/digitalgarage/course/effective-networking/lesson/194" desc="A quick video on developing interpersonal skills" button="Check it out!"></grid-1-x-2>
+
+### [Click here to check out the past exams!](https://exams-library-utoronto-ca.myaccess.library.utoronto.ca/simple-search?query=title%3Acsc290)
```

Comment:
This doesn't render although I'm not sure why its here (Might want to double check to make sure that this isn't here) (i.e. It should be at the top rather than at the bottom)